00-HTML-CSS-basics/HTML/Excersise2/form.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE HTML>
+<html>
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="autor" content="Demian-Moschin">
+    <meta name="description" content="Submit form for web developers">
+    <meta name="viewport" content="width=device-width, user-scalable=no,initial-scale=1" >
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" >
+
+    <title>Form completition</title>
+
+  </head>
+
+  <body>
+
+    <form id="sign-form" accept-charset="UTF-8">
+
+      <fieldset>
+        <legend>Personal information: </legend>
+        <p>
+         <label>First Name:<input form="sign-form" autofocus name="firstName" required size="20" maxlength="20" placeholder="e.g.: John" /></label>
```

Comment:
The form attribute is not suported by all the browsers, I think it's better to wrap all the input fields inside the form tag. Also the name attribute and ids attributes are better defined like `first-name` insstead of `firstName`.